docs/source/markdown/options/cpus.container.md

```diff
@@ -5,7 +5,7 @@
 #### **--cpus**=*number*
 
 Number of CPUs. The default is *0.0* which means no limit. This is shorthand
-for **--cpu-period** and **--cpu-quota**, so you may only set either
+for **--cpu-period** and **--cpu-quota**, so specify either
```

Comment:
Not your fault (not your wording), but this sentence is hard to parse. Would you mind cleaning it up a little? 
>This is shorthand for [...] and cannot be set together
>This is shorthand for [...] and cannot be specified on the same command line
>This is shorthand for [...]. It is an error to combine **--cpus** with either of those.

Reply:
I'm not real clear what you're suggesting here @edsantiago, but I like the middle one of these three.